jaeger-core/src/test/java/io/jaegertracing/ConfigurationTest.java

```diff
@@ -85,7 +85,19 @@ public void clearProperties() throws NoSuchFieldException, IllegalAccessExceptio
   public void testFromEnv() {
     System.setProperty(Configuration.JAEGER_SERVICE_NAME, "Test");
     assertNotNull(Configuration.fromEnv().getTracer());
-    assertFalse(GlobalTracer.isRegistered());
```

Comment:
If removing from here, then should remove GlobalTracer completely from this test class (i.e. clearProperies()).

Reply:
Done. I removed it because I find it confusing that the test would make a relationship between `fromEnv()` and the `GlobalTracer`. If there's indeed a need to ensure the global tracer is not registered after `fromEnv()` is called, I can put it back.